src/main/resources/META-INF/rewrite/spring-boot-30.yml

```diff
@@ -53,7 +53,6 @@ recipeList:
   - org.openrewrite.gradle.UpdateGradleWrapper:
       version: ^7.4
       addIfMissing: false
-  - org.openrewrite.java.migrate.jakarta.JavaxMigrationToJakarta
```

Comment:
Hmm; might this move now introduce dependencies that would otherwise be available transitively?